proto/tendermint/abci/types.proto.intermediate

```diff
@@ -0,0 +1,472 @@
+syntax = "proto3";
+package tendermint.abci;
+
+// For more information on gogo.proto, see:
+// https://github.com/gogo/protobuf/blob/master/extensions.md
+import "tendermint/crypto/proof.proto";
+import "tendermint/types/types.proto";
+import "tendermint/crypto/keys.proto";
+import "tendermint/types/params.proto";
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+// This file is copied from http://github.com/tendermint/abci
```

Comment:
Is this right? I'm not seeing what file it's copied from.

I was going to ask whether we could have the script make this copy, if it really is, but now I'm not sure where this text came from.

Reply:
Ah, this is, it appears, in the current `abci/types.proto` file, which I copied to make this one. These `.intermediate` files are my attempt at creating a `types.proto` file that can build with current master. I effectively copy+pasted from the various abci++ files and branches to get this here. A script to make _this_ file wouldn't make sense since this file is a hodge-podge of changes that don't appear to actually exist together in one place.

Reply:
My hope is that, once the ABCI++ project is over, we can delete these `.intermediate` files.

Reply:
I see—so this file is really a palimpsest of a file that was itself copied from there. Yeah, I'm fine with leaving these things in a temporarily-intermediate state to get ourselves out of the quagmire.

I might consider deleting these comments, since they're already obsolete and probably just add confusion. It might be worth injecting a comment at the top to the effect of "Don't modify this file. This is a temporary patch for the migration to ABCI++. See issue xyz. It will be removed after pdq."

Reply:
Added some comments to clarify and opened https://github.com/tendermint/tendermint/issues/8066 to track their removal.